main-cookbooks/teracy-dev/recipes/normalization.rb

```diff
@@ -0,0 +1,40 @@
+#
+# Author:: Hoat Le <hoatle@teracy.com>
```

Comment:
@hieptranquoc please add you as "author" here

Reply:
fixed!